BUILD.bazel

```diff
@@ -115,9 +70,8 @@ cmake(
             "libcurl.lib",
         ],
     }),
-    visibility = ["//visibility:private"],
     deps = [
-        "//:boringssl_lib",
+        "@boringssl_with_bazel_rules_curl_build_file//:boringssl_lib",
```

Comment:
What happened to the comment we had worked on together that explained why we were doing this whole dance? Let's please put it in!

Reply:
Fixed